sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -0,0 +1,537 @@
+# Authors:
+#
+#          Giorgio Patrini
+#
+# License: BSD 3 clause
+from __future__ import division
+
+import re
+
+import numpy as np
+from scipy import sparse
+
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_raises
+from sklearn.utils.testing import assert_raises_regex
+from sklearn.utils.testing import assert_allclose
+from sklearn.utils.testing import ignore_warnings
+from sklearn.utils.testing import assert_warns
+from sklearn.utils.testing import assert_warns_message
+from sklearn.utils.testing import assert_no_warnings
+from sklearn.utils.testing import SkipTest
+
+from sklearn.preprocessing._encoders import _transform_selected
+from sklearn.preprocessing.data import Binarizer
+from sklearn.preprocessing._encoders import OneHotEncoder
+from sklearn.preprocessing._encoders import OrdinalEncoder
+
+from sklearn import datasets
+
+iris = datasets.load_iris()
+
+# Make some data to be used many times
+rng = np.random.RandomState(0)
+n_features = 30
+n_samples = 1000
```

Comment:
Not convinced there's any benefit to constants this big...

Reply:
whoops, more leftovers from copying it out of `test_data.py`, removing